src/javascript/app_2/Stores/Modules/Trading/Helpers/proposal.js

```diff
@@ -36,11 +36,20 @@ export const createProposalRequests = (store) => {
 
 const createProposalRequestForContract = (store, type_of_contract) => {
     const obj_expiry = {};
+
     if (store.expiry_type === 'endtime') {
+        const server_time = store.root_store.common.server_time;
+        const start_date  = moment.unix(store.start_date || server_time).utc();
```

Comment:
@cakasuma `moment.unix()` return an invalid object. it should be
```suggestion
        const start_date  = moment.utc(store.start_date || server_time).unix();
```

Reply:
@cakasuma ignore my last comment. Your implementation is also correct. since `moment.unix(xyz)` returns a moment object.